pulpcore/cli/common.py

```diff
@@ -45,19 +48,42 @@ def wait_for_task(self, task_href, timeout=120):
         raise click.ClickException("Task timed out")
 
 
+def _config_callback(ctx, param, value):
+    xdg_config_home = os.path.expanduser(os.environ.get("XDG_CACHE_HOME") or "~/.config")
```

Comment:
Will this only be called if `--config` is specified?

Reply:
It's always called. I'll move it down to the `else`.

Reply:
Sorry, i meant the whole function. And i was wondering if the defaults from the default location are read if you didn't use the `--config` parameter.

Reply:
Ah yea, that's correct. This is always called and if there's no value, it'll use the default location.